shimmingtoolbox/read_nii.py

```diff
@@ -0,0 +1,76 @@
+import json
+import math
+import nibabel as nib
+import numpy as np
+import os
+
+
+def read_nii(nii_path, return_image=False):
+    """ Reads a nifti file and returns the corresponding image and info. Also returns the associated json data.
+    Args:
+        nii_path (str): direct path to the .nii or .nii.gz file that is going to be read
+        return_image (:obj:`bool`, optional): Defines if the function should (True) or should not (False) return `image`
+    Returns:
+        info (Nifti1Image): Objet containing various data about the nifti file (returned by nibabel.load)
+        json_data (dict): Contains the different fields present in the json file corresponding to the nifti file
+        image (ndarray): Image contained in the read nifti file. Siemens phase images are rescaled between 0 and 2pi.
+    """
+
+    info = nib.load(nii_path)
```

Comment:
The load function of Nibabel returns an object with the image and the header of the nii file. If you only, want to return the header, you should then add:
```suggestion
    info = nib.load(nii_path).header
```

Reply:
I guess that's open to discussion. Indeed we will mainly use the header data but we might sometimes want to have the whole object (not aware of such case yet however). I don't really know what would be the best option but I admit that it would be easier to play with the data if it was only the header content. @po09i do you have an opinion on this ?

Reply:
let's not spend too much time overthinking this please-- the project is currently at <5% of its final implementation-- there is still a LOT of code to move from matlab to python, so a lot of APIs will get refactored, etc. for now, just pass the nibabel object because it contains everything-- in the future if we think it makes more sense to pass something else, we will do it.